src/mgr/ActivePyModules.cc

```diff
@@ -591,7 +601,8 @@ void ActivePyModules::notify_all(const std::string &notify_type,
     // C++ code, and avoid any potential lock cycles.
     dout(15) << "queuing notify (" << notify_type << ") to " << name << dendl;
     // workaround for https://bugs.llvm.org/show_bug.cgi?id=35984
```

Comment:
Should this be directly above line 605 since it refers to an issue with lambdas

Reply:
Done